styles.css

```
@@ -687,6 +686,49 @@ body {
   display: none;
 }
 
+/* RESPONSIVE MENU */
+
+.menu-toggle {
+  display: none;
+}
+
+.menu-toggle.show {
+  position: absolute;
+  z-index: 9999;
+  background-color: #6070ff;
+  width: 375px;
+  height: 812px;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(8px);
+  display: block;
+}

```

```suggestion
.menu-toggle.show {
  position: fixed;
  z-index: 9999;
  background-color: #6070ff;
  width: 100%;
  height: 812px;
  mix-blend-mode: multiply;
  backdrop-filter: blur(8px);
  display: block;
  overflow-y: hidden;
}
```

- [ ] As mentioned by the previous reviewer, your mobile menu container should cover the screen width as in the Figma design. You can fixed that by giving `width:100%` property and to avoid the menu scroll with the page, kindly add `position: fixed;` and `overflow-y: hidden;` properties to the menu container. That will make you mobile menu look identical to the Figma design.